storage/sql/migrate.go

```diff
@@ -77,8 +79,8 @@ var migrations = []migration{
 				public boolean not null,
 				name text not null,
 				logo_url text not null
-			);
-		
+			);`,
```

Comment:
[nit] If we do this as multiple calls to `tx.Exec`, we don't _need_ the ending `;`, I think.